conversion/conversion_from_source.go

```diff
@@ -289,11 +287,12 @@ func (sads *DataFromSourceImpl) dataFromDatabase(ctx context.Context, sourceProf
 			}
 
 			monitoringResources := metrics.MonitoringMetricsResources{
-				ProjectId:            targetProfile.Conn.Sp.Project,
+				MigrationProjectId:   migrationProjectId,
 				DataflowJobId:        dfOutput.JobID,
 				DatastreamId:         streamingCfg.DatastreamCfg.StreamId,
 				JobMetadataGcsBucket: gcsBucket,
 				PubsubSubscriptionId: streamingCfg.PubsubCfg.SubscriptionId,
+				SpannerProjectId:     targetProfile.Conn.Sp.Project,
```

Comment:
why would monitoring resource need spanner project id? Does it try to pull metrics from cross proejct? how would auth handled?

Reply:
Monitoring resource object contains details of all the resources created for a migration. I added SpannerProjectId here for completeness. How auth and permissions needs to be handled will need to be taken up as a separate task.